sklearn/tests/test_docstring_parameters.py

```diff
@@ -45,7 +45,8 @@
 
 # numpydoc 0.8.0's docscrape tool raises because of collections.abc under
 # Python 3.7
-@pytest.mark.filterwarnings('ignore::DeprecationWarning')
+@pytest.mark.filterwarnings(
```

Comment:
if this is about numpydoc, why would we change it?

Reply:
We don't error anymore on `DeprecationWarning`, so I should remove the comment about numpydoc (still open to discussion of course, so I won't right now)

but we still need to ignore FutureWarning because some entire modules are deprecated.

Reply:
I know, but we'd need to fail on a `DeprecationWarning`accidentally raised by us, or by our dependencies (not sure if we do that now though)